### PR TITLE
Created check_anagrams.py in strings

### PR DESCRIPTION
**Pull request by Kush1101**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Kush1101** [comment]
@cclauss. I need you insights.

File: strings/check_anagrams.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def check_anagrams(string_A: str, string_B: str) -> bool:
```
Comment (cclauss) [inline strings/check_anagrams.py:1]:
```suggestion
def check_anagrams(a: str, b: str) -> bool:
```

Comment (Kush1101) [reply strings/check_anagrams.py:1]: Working on it

File: strings/check_anagrams.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    Two strings are anagrams if they are made of the same letters
+    arranged differently (ignoring the case).
+    >>> check_anagrams('Silent','Listen')
+    True
+    >>> check_anagrams('This is a string','Is this a string')
+    True
+    >>> check_anagrams('There','Their')
+    False
+    """
+    sorted_list_A = sorted(string_A.lower())
```
Comment (cclauss) [inline strings/check_anagrams.py:12]:
```suggestion
    return sorted(a.lower()) == sorted(b.lower())
```

**Comment by Kush1101** [comment]
@cclauss I made all the changes.

**Comment by Kush1101** [comment]
@cclauss I made all the changes. Please review it.

**Comment by Kush1101** [comment]
@cclauss can this be merged?

**Review by cclauss — APPROVED** [review]

**Comment by Kush1101** [comment]
Thanks!

**Pull request merged**